code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -133,7 +147,7 @@
 		return
 	PoolOrNew(/obj/effect/overlay/temp/guardian/phase/out, get_turf(src))
 	unbuckle_mob(force=1)
-	loc = summoner
+	forceMove(summoner)
```

Comment:
This is something I forgot to do with the previous forcemove changes, should have the same result as currently with less lighting bugs.